e2e/kubernetes/resource_kubernetes.go

```diff
@@ -0,0 +1,666 @@
+package kubernetes
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"regexp"
+	"strings"
+
+	"github.com/e2eterraformprovider/terraform-provider-e2e/client"
+	"github.com/e2eterraformprovider/terraform-provider-e2e/models"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+func ResourceKubernetesService() *schema.Resource {
+	return &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"name": {
```

Comment:
Keeping a ForceNew in name and version of K8 allows user to create a new k8 resource with change name and version deleting the previous one, a user can see that as an update operation but it will be a delete recreate. Please check this.